### DISCUSSION TITLE
Add return button to non-scrollable panel in GitHub actions

### DISCUSSION THREAD

**Discussion by pmcpg** [community/community#61811] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi all,

I noticed a behavior when using GitHub actions, which I'd like to discuss:

We start in the overview of runs:

![grafik](https://github.com/community/community/assets/101276471/dd11c979-9d17-4bb1-a360-38606c4b49a3)

When we click on one individual run:

![grafik](https://github.com/community/community/assets/101276471/4160aad6-59d4-4be5-86cc-3b25c9cef529)

We can always go back to the aforementioned overview with this button marked in red rectangle.

### Problem

![grafik](https://github.com/community/community/assets/101276471/2b00dc56-d6e5-4ace-8d9c-30c8156dd378)

When we scroll down in logs of a run, the _back to overview_ button disappears and only the run overview is visible. I know you can click on browser's _back_ button, but if we clicked on _Usage_ or _Workflow file_ on the menu first, we won't go back to the root overview. So we need to scroll to the top of the page, so the _back to overview_ button is visible again. Especially when constructing pipelines this is super annoying 😢 

### Solution

Add the _back to overview_ to the non-scrollable part so even when we are at the bottom of the site we can go back to overview with one click. Thank you and of course open for discussions.

**Comment by moonfury-ops** [comment]
Thank you

**Comment by Pyrulethe4ool** [comment]
Love.ini

**Comment by Pearce-Ropion** [comment]
Alternatively, add a button to quickly scroll back to the top.

**Comment by raaymax** [comment]
yes! please add this, or scroll to the top 🙏
